src/client/lib/ErrorLink.ts

```diff
@@ -16,12 +15,7 @@ const DEFAULT_ERROR_LINK: ErrorLink = {
   linkText: linkText.DEFAULT,
 };
 
-const CUSTOM_ERROR_LINKS = new Map<string, ErrorLink>([
-  [
-    ChangePasswordErrors.PASSWORD_NO_MATCH,
-    { link: locations.HELP, linkText: linkText.PASSWORD },
-  ],
-]);
+const CUSTOM_ERROR_LINKS = new Map<string, ErrorLink>([]);
```

Comment:
more of a question than a change:
in #42 we added this functionality in to customise the link shown in the global error

It seems that this PR removed the global error, in turn the link to the password faqs, I wonder if we need to add a link somewhere to those faqs?

think that might be a question for product

Reply:
I will ask

Reply:
it's strange that this was returned only when the passwords do not match. The information on the FAQ doesn't seem relevant to the reset form